go/vt/vttablet/sandboxconn/sandboxconn.go

```diff
@@ -55,6 +55,9 @@ type SandboxConn struct {
 	MustFailStartCommit         int
 	MustFailSetRollback         int
 	MustFailConcludeTransaction int
+	// MustFailExecute is keyed by the statement type and stores the number
+	// of errors to return for that statement type.
+	MustFailExecute map[sqlparser.StatementType]int
```

Comment:
The name is a little confusing considering the comment

Reply:
it is similar to the naming convention of other MustFails in the struct.

Reply:
added more comment for clarity

Reply:
okay got it